app/scripts/components/exploration/components/datasets/tile-link-modal.tsx

```diff
@@ -50,7 +50,8 @@ export function TileUrlModal(props: {
         <Form>
           {[
             { label: 'XYZ Tile URL', value: tileUrls?.xyzTileUrl },
-            { label: 'WMTS Tile URL', value: tileUrls?.wmtsTileUrl }
+            { label: 'WMTS Tile URL', value: tileUrls?.wmtsTileUrl },
+            { label: 'WMS Tile URL', value: tileUrls?.wmsTileUrl }
```

Comment:
fyi that `raster-paint-layer` doesn't handle these metadata well. The fix is currently opened in this PR: https://github.com/NASA-IMPACT/veda-ui/pull/1607 (I will follow up once this PR is merged)